lxd/storage/drivers/driver_pure_util.go

```diff
@@ -507,3 +531,33 @@ func (d *pure) serverName() (string, error) {
 
 	return hostname, nil
 }
+
+// getVolumeName returns the fully qualified name derived from the volume's UUID.
+func (d *pure) getVolumeName(vol Volume) (string, error) {
```

Comment:
can we add tests for this sort of well-isolated function? Would help get our coverage rates up, especially for code we cannot easily test in the CI system tests